Vagrantfile

```diff
@@ -77,6 +77,17 @@ def mount_folders(config, mount_options)
 end
 
 Vagrant.configure(2) do |config|
+  config.vm.boot_timeout=600
```

Comment:
This is already done on line 94 and can be removed.

Reply:
which version's line 94? I didn't see that in the main branch or in my branch

Reply:
Sorry, meant line 84:

https://github.com/Submitty/Submitty/blob/4a6309822740858cab5d1052768fd33815ea0aec/Vagrantfile#L84

Reply:
OK, I see. Will modify it soon. Thanks